### DISCUSSION TITLE
Developer Documentation: Light mode or dark more?

### DISCUSSION THREAD

**Discussion by ielrekaa** [deepgram/community#37] [General help]
Do you prefer light mode or dark mode for Docs?

**Comment by BekahHW** [comment]
I'm mostly into dark mode, but I have astigmatism, so sometimes I get the halation effect. I've found that I do better with dark purples, rather than black backgrounds. I don't mind light mode though.

**Comment by jviso** [comment]
A first-class toggle on the site is the best, I think!

**Reply by michaeljolley** [reply]
A good toggle that defaults to system preferences is :chefskiss:

**Comment by michaeljolley** [comment]
I normally work in dark mode. It's my system preference. But I do appreciate when I have the option to switch to light mode for the same reason @BekahHW pointed out.

**Comment by adeschamps** [comment]
Whatever the individual preference, supporting the [`prefers-color-scheme`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) CSS feature would be nice.

**Comment by bowlofeggs** [comment]
I've gone to lengths to make sure my computer doesn't blind me.

**Comment by JesseRWeigel** [comment]
I think it is important to have both. If only dark mode, then it is unreadable in sunlight. Light mode is painful to look at in a dark room and gives me headaches if I look at it to long.
